tests/suite/test_v_s_route_status.py

```diff
@@ -218,19 +217,18 @@ def test_status_invalid_vs(
         self.patch_valid_vs(kube_apis, v_s_route_setup)
         assert (
             response_m["status"]
-            and response_m["status"]["reason"] == "Rejected"
+            and response_m["status"]["reason"] == "NoVirtualServerFound"
```

Comment:
@pleshakov should this be "Ignored" or "NoVirtualServerFound" ?

Reply:
This is for when this `VSR` is not referenced by any `VS` right? 

Then `NoVirtualServerFound` but let's wait for @pleshakov to be 100% sure.